core/serve.go

```diff
@@ -45,12 +46,14 @@ func Serve(path string, options ServeOptions) error {
 	done := make(chan bool)
 	rebuildCh := make(chan string)
 
+	fmt.Println(theme.GeneratedDir(filepath.Join(path, config.ThemesDir), cfg.Theme))
```

Comment:
Is this intentional? :)

Reply:
no

Reply:
Should I remove it or are going to do that yourself?

Reply:
I haven't much time till Monday, so you can do it.

Reply:
removed it